packages/mock-block-dock/dev/dev-app.tsx

```diff
@@ -9,101 +9,88 @@ import { TestReactBlock } from "./test-react-block";
 
 const node = document.getElementById("app");
 
+type TestBlockType =
+  | "react"
+  | "custom-element"
+  | "html-at-url"
+  | "html-as-string";
+
 const DevApp: FunctionComponent = () => {
-  const [entityId, setEntityId] = useState(1);
-  const [name, setName] = useState("World");
-  const [readonly, setReadonly] = useState(false);
+  const [testBlockType, setTestBlockType] = useState<TestBlockType>("react");
 
   const blockEntity = {
-    entityId: `test-entity-${entityId}`,
+    entityId: `test-entity-1`,
     entityTypeId: "test-type-1",
-    properties: { name },
+    properties: { name: "World" },
   };
 
+  let blockDefinition;
+
+  switch (testBlockType) {
+    case "custom-element":
+      blockDefinition = {
+        customElement: {
+          elementClass: TestCustomElementBlock,
+          tagName: "test-custom-element-block",
+        },
+      };
+      break;
+
+    case "html-at-url":
+      blockDefinition = {
+        html: {
+          url: "./test-html-block/block.html",
+        },
+      };
+      break;
+
+    case "html-as-string":
+      blockDefinition = {
+        html: {
+          source: testBlockString,
+          url: new URL(
+            "./test-html-block/block.html",
+            window.location.toString(),
+          ).toString(),
+        },
+      };
+      break;
+
+    case "react":
+    default:
+      blockDefinition = {
+        ReactComponent: TestReactBlock,
+      };
+  }
+
   return (
-    <div style={{ fontFamily: "sans-serif" }}>
-      <div
-        style={{
-          border: "1px solid black",
-          padding: 15,
-        }}
-      >
-        <h2>Update component props from outside the block</h2>
-        <button type="button" onClick={() => setEntityId(entityId + 1)}>
-          Increment Entity ID
-        </button>
-        {/* eslint-disable-next-line jsx-a11y/label-has-associated-control -- https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/869 */}
-        <label style={{ marginLeft: 20 }}>
-          <input
-            type="checkbox"
-            onChange={(evt) => setReadonly(evt.target.checked)}
-          />
-          Read only mode
+    <>
+      <div style={{ position: "absolute", right: 16, top: 16, zIndex: 100 }}>
+        {/* eslint-disable-next-line jsx-a11y/label-has-associated-control -- https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/869  */}
```

Comment:
We can avoid this disable directive by [configuring](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/label-has-associated-control.md) the rule:

```json
{
  "rules": {
    "jsx-a11y/label-has-associated-control": [ "error", {
      "assert": "either",
    }],
  }
}
```

Reply:
Updated here => [876a2aa](https://github.com/blockprotocol/blockprotocol/pull/507/commits/876a2aacd2e5cdda39293b3cbfa6fc8227c03b7b)

Reply:
Try running `yarn fix:eslint` — this should make CI green

Reply:
Done